### PR TITLE
New more info light

### PR DESCRIPTION
**Pull request by piitaya**
## Proposed change

CleanShot 2023-01-24 at 13 36 00

CleanShot 2023-01-24 at 13 36 08

CleanShot 2023-01-24 at 13 36 19

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by bramkragten** [comment]
Can we make the height of the color picker the same as the sliders (add padding to top/bottom) so the tab bar/dialog doesnt jump around?

File: package.json [hunk 1]
```diff
@@ -71,6 +71,7 @@
     "@material/mwc-textfield": "^0.27.0",
     "@material/mwc-top-app-bar-fixed": "^0.27.0",
     "@material/top-app-bar": "=14.0.0-canary.53b3cad2f.0",
+    "@material/web": "^1.0.0-pre.1",
```
Comment (bramkragten) [inline package.json:74]: Unused?

File: src/dialogs/more-info/controls/more-info-light.ts [hunk 2]
```diff
@@ -1,233 +1,116 @@
...
-  CSSResultGroup,
-  html,
-  LitElement,
-  PropertyValues,
-  TemplateResult,
-} from "lit";
-import { customElement, property, state } from "lit/decorators";
-import memoizeOne from "memoize-one";
+import "@material/web/iconbutton/outlined-icon-button";
+import "@material/web/iconbutton/outlined-icon-button-toggle";
```
Comment (bramkragten) [inline src/dialogs/more-info/controls/more-info-light.ts:3]: Unused?

**Comment by bramkragten** [comment]
When a color is set, and not a CT, the indicator of the CT slider is at the bottom, can we hide the indicator when there is not CT value set?

File: src/dialogs/more-info/controls/more-info-light.ts (hunk 2 above)
Comment (piitaya) [reply src/dialogs/more-info/controls/more-info-light.ts:3]: Yes I will remove it

**Review by bramkragten — APPROVED** [review]

**Pull request merged**